content/pt-br/docs/setup/best-practices/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Best practices
```

Comment:
@brbernardo Would it be good to translate this too? 😊

https://deploy-preview-31743--kubernetes-io-main-staging.netlify.app/pt-br/docs/setup/best-practices/

![image](https://user-images.githubusercontent.com/46767780/154385845-fc0353f0-43a2-4d48-9ae7-93d865c7cb25.png)